source/miscellaneous.rst

```diff
@@ -0,0 +1,71 @@
+Assembling an Executable Jar file
+===================================
+
+Introduction
+------------
+
+This page highlights how you can make a ``jar`` file from any of the models that you create. A ``jar`` file is an executable java file that can be used to run your code
+
+Steps for creating and executing a new jdk jar file
+----------------------------------------------------
+
+1.  Go to the sbt shell in IntelliJ, and type ``assembly``
+
+.. figure:: _static/images/jar-doc-1.png
```

Comment:
Screenshot this image again, please. Go through my comments on the Slack channel on how to make the images neat.

1. Don't use ``BharatSim-master`` as the folder name, it's likely to confuse beginners
2. Everything above the ``sbt shell`` in this figure is unimportant and distracting